src/main/java/duke/Time.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exception.DukeException;
+
+/**
+ * Time class is used to parse the date and time from
+ * {@code String} to {@code LocalDateTime} or {@code LocalDate} object.
+ */
+public class Time {
```

Comment:
I like how you have created a separate class for Time in order to handle formatting issues.